.github/workflows/test-all-packages.yml

```diff
@@ -1,6 +1,10 @@
 name: Test all Packages
 
-on: [pull_request]
+on:
+ push:
+   branches: [master]
+ pull_request:
```

Comment:
I think we want to have pull_request on every branch, not just master

Reply:
ok let's see if that change has the right syntax